tests/cecil-tests/TaskAssemblyTests.cs

```diff
@@ -40,6 +40,8 @@ bool IsTaskType (TypeDefinition? td)
 		[Test]
 		public void EnsureOnlyCodeInBaseTasks ()
 		{
+			if (assembly.Contains ("Xamarin.Mac.Tasks.dll"))
```

Comment:
This should check `Configuration.include_mac` and Assert.Ignore if true. That way we don't have to remember to turn it back on later.